src/VirtoCommerce.NotificationsModule.Data/Migrations/20190528144557_InitialNotifications.cs

```diff
@@ -224,15 +231,18 @@ FROM INFORMATION_SCHEMA.TABLES
 	                            null, null
                             FROM 
                             [PlatformNotificationTemplate] pnt
-                            INNER JOIN 
-                            (
-	                            SELECT [NotificationTypeId], MAX([Id]) [Id] FROM [PlatformNotificationTemplate]
-	                            GROUP BY [NotificationTypeId]
-                            ) pnt2
+                            INNER JOIN pnt2

```

Comment:
Why do you join pnt2 there ? I don't see link on pnt2 in the select

Reply:
Just to define notification ID as MAX from template ids.